test/onnx/test_utility_funs.py

```diff
@@ -602,6 +601,36 @@ def test_error_on_data_parallel(self):
                                     "unwrap model from torch.nn.DataParallel. Try "):
             torch.onnx.export(model, x, f, opset_version=self.opset_version)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_loop_sequence_dim(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                l = [x, y]
+                return l
+
+        model = Module()
+
+        x = torch.randn(2, 3)
+
+        # Case 1: varying dimension size.
```

Comment:
I think this comment is referring to setting `dynamic_axes` in the export call, if so make it clearer:
```suggestion
        # Case 1: dynamic axis
```

Reply:
Did you forget to make this change?